### PR TITLE
[onert] Remove unused codes in gpu_cl

### PR DESCRIPTION
**Pull request by YiHyunJin**
This will remove unused codes in gpu_cl

Signed-off-by: hj0412-yi <hj0412.yi@samsung.com>

### CODE REVIEW

**Comment by YiHyunJin** [comment]
Related : #7123

File: runtime/onert/backend/gpu_cl/open_cl/kernels/Util.cc [hunk 1]
```diff
@@ -77,132 +77,133 @@ std::string GetCommonDefines(CalculationsPrecision precision)
...
+//   for (const auto &wg : wgs)
+//   {
+//     const int wg_size = wg.x * wg.y * wg.z;
+//     if (wg_size <= max_wg_size)
+//     {
+//       return wg;
+//     }
+//   }
+//   return {1, 1, 1};
+// }
```
Comment (chunseoklee) [inline runtime/onert/backend/gpu_cl/open_cl/kernels/Util.cc:102]: Any reason for commenting this w/o removing it ?

Comment (YiHyunJin) [reply runtime/onert/backend/gpu_cl/open_cl/kernels/Util.cc:102]: Deleted codes are unnecessary codes or will be used as a whole.
But this code will be partially used. So it is commented

Comment (seanshpark) [reply runtime/onert/backend/gpu_cl/open_cl/kernels/Util.cc:102]: How about useing `#if 0 ... #endif` ?

File: runtime/onert/backend/gpu_cl/open_cl/Operations.cc [hunk 2]
```diff
@@ -260,18 +262,16 @@ BHWDC CalculateOutputShape(const BHWDC &input, const DepthwiseConvolution3DAttri
                  attr.weights.shape.get<Axis::INPUT_CHANNELS>());
 }
 
-BHWC CalculateOutputShape(const BHWC &input, const SliceAttributes &attr)
+BHWC CalculateOutputShape(const BHWC &, const SliceAttributes &attr)
```
Comment (seanshpark) [inline runtime/onert/backend/gpu_cl/open_cl/Operations.cc:265]: I think this change and others like are not needed from using `#if 0`

File: runtime/onert/backend/gpu_cl/open_cl/kernels/Util.cc [hunk 3]
```diff
@@ -101,7 +103,8 @@ int3 GetFirstSuitableWorkGroup(const std::vector<int3> &wgs, int max_wg_size)
   return {1, 1, 1};
 }
 
-int GetRecommendedBlockSizeForConv(const DeviceInfo &device_info, CalculationsPrecision precision,
+int GetRecommendedBlockSizeForConv(const DeviceInfo &device_info, CalculationsPrecision
+precision,
```
Comment (seanshpark) [inline runtime/onert/backend/gpu_cl/open_cl/kernels/Util.cc:107]: This change seems unintended...

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**